src/main/webapp/main.js

```diff
@@ -30,3 +32,47 @@ Promise.all([mapPromise, domPromise]).then(() => {
 function initMap() {
   myMap = new ChapMap();
 }
+
+
+connection.onopen = () => {
+};
+
+connection.onclose = () => {
+};
+
+connection.onerror = (error) => {
+  console.error(error);
+};
+
+connection.onmessage = (event) => {
+  let li = document.createElement('li');
```

Comment:
nit: This only needs to be created inside the else statement right before you use it to avoid creating unnecessary DOM elements.